tests/test_ios_testflight_external_distribution.py

```
@@ -0,0 +1,85 @@
+        "id": group_id,
+        "name": name,
+        "is_internal": is_internal,
+        "has_access_to_all_builds": has_access_to_all_builds,
+    }
+
+
+def main():
+    module = _load_module()
+
+    groups = [
+        _group("internal-1", "cmux beta", True),
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)